src/components/learning-guide-chart.coffee

```diff
@@ -1,14 +1,14 @@
 _ = require 'underscore'
 d3 = require 'd3'
+{AppConfigStore} = require '../flux/app-config'
 
-CLOUD_PATH = '/style/resources/learning-guide/cloud.svg'
-PLANE_PATH = '/style/resources/learning-guide/openstax-plane.svg'
-CITYSCAPE_PATH = '/style/resources/learning-guide/houston-skyline.svg'
-
-FLAG_BLUE = '/style/resources/learning-guide/flag-blue.svg'
-FLAG_GREEN = '/style/resources/learning-guide/flag-green.svg'
-FLAG_YELLOW = '/style/resources/learning-guide/flag-yellow.svg'
-FLAG_GREY = '/style/resources/learning-guide/flag-grey.svg'
+CITYSCAPE_PATH = 'learning-guide/houston-skyline.svg'
```

Comment:
so far all the SVG files in the `dist/archive.tar.gz` (`gulp prod`) end up in the root dir. We should probably change the gulpfile.